lib/pgBackRest/Backup/Backup.pm

```diff
@@ -436,7 +440,7 @@ sub processManifest
 
         # A keep-alive is required here because if there are a large number of resumed files that need to be checksummed
         # then the remote might timeout while waiting for a command.
-        $oProtocolMaster->keepAlive();
+        protocolKeepAlive();
```

Comment:
This is a nice wrapper function that means we don't have to worry about whether there is a remote or not.  It also simplifies test cases.